### ISSUE TITLE
🐛 BUG: globEager is not a function error when fetchContent is preceded by a string containing an apostrophe

### ISSUE THREAD

**Issue by JoelHobson** [snowpackjs/astro#976]
### What package manager are you using?

npm

### What operating system are you using?

Windows 10

### Describe the Bug

I'm attempting to render the following page:

```
---
let someVariable = "my name's blog";
let allPosts = Astro.fetchContent('./posts/*.md');
---
Some text
```

I get this error from the dev server:
```
TypeError: __import_meta.globEager is not a function
    at Object.__render (/_astro/src/pages/index.astro.js:35:49)
    at Object.__renderPage (/_astro/src/pages/index.astro.js:61:46)
    at load (file:///D:/Projects/astrotest/node_modules/astro/dist/runtime.js:165:34)
    at async Server.<anonymous> (file:///D:/Projects/astrotest/node_modules/astro/dist/dev.js:21:20)
```

This appears to be caused by the string containing an odd number of apostrophes appearing above the call to `Astro.fetchContent`. 

I can cause the page to render by commenting out the `Astro.fetchContent` line, or by moving that line above the `someVariable` declaration, or by removing the apostrophe (or adding an additional one) from the string in `someVariable`. Escaping the apostrophe `\'` has no effect.

### Steps to Reproduce

1. `npm init astro` using template Starter Kit (Generic)
2. Replace the contents of `pages/index.astro` with the template from above
3. Run the dev server and access the index page.
4. A 500 error is shown, instead of the content in the page.

**Comment by mmarkelov** [comment]
This is weird. There is no error with this:
```
let someVariable = "my name&#39;s blog";
let allPosts = Astro.fetchContent('./posts/*.md');
```

**Comment by matthewp** [comment]
I'm also unable to recreate. @JoelHobson could you try again and let us know with a newer version of Astro? It's possible you ran into a bug that's since been fixed.

**Comment by tusharsadhwani** [comment]
@matthewp  I can still reproduce this, with astro v0.19.1

**Comment by tusharsadhwani** [comment]
Here's my frontmatter for `index.astro`:

```astro
---
import BaseHead from '../components/BaseHead.astro';
import BlogHeader from '../components/BlogHeader.astro';
import BlogPostPreview from '../components/BlogPostPreview.astro';

let title = 'Tushar\'s Blog';
let description = 'My thoughts and learnings.';

let allPosts = Astro.fetchContent('./posts/*.md');
allPosts = allPosts.sort((a, b) => new Date(b.publishDate) - new Date(a.publishDate));
---
```

It crashes. Adding another parameter after `description`, or removing the apostrophe in `title` fixes it.

**Comment by jasikpark** [comment]
What happens if you use `let title = "Tushar's Blog";` or 
```
let title = `Tushar's Blog`;
```

**Comment by tusharsadhwani** [comment]
@jasikpark Nothing helps. Escaping the apostrope or not, using whatever quotes. As long as there's the apostrophe, it'll crash.

**Comment by tusharsadhwani** [comment]
This is actually true for the `description` field as well.
BUT interestingly, this works:

```js
let title = "Tushar's Blog";
let description = 'My thoughts and learnings.//';
```
This works too:
```js
let title = "Tushar's Blog//";
let description = 'My thoughts and learnings.';
```

---

What is seems like, is that the apostrophe is causing an escape somewhere, and then the `//` is causing the problem to get commented out.
Maybe this is why this bug is rare? Because in the default blog template, there's another variable `permalink = 'https://example.com` which has the `//` in it, causing the bug to hide.

**Comment by tusharsadhwani** [comment]
@jasikpark @matthewp hopefully this will narrow down the bug enough :smile:

**Comment by jasikpark** [comment]
https://codesandbox.io/s/optimistic-river-0uiju?file=/src/pages/index.astro here's a replication of this bug on 0.19.3

**Comment by matthewp** [comment]
Awesome, thanks for narrowing this down!

**Comment by philipp-tailor** [comment]
I ran into the same issue, and could help reproduce, if needed.

**Comment by jadonn** [comment]
I had this same issue. This discussion here helped me get passed it on version 0.20.12. I just moved my string with an apostrophe/single quote to after my call for Astro.fetchContent, and the error went away.

This made the error happen:
```
const title = "Someone's blog"
let allPosts = Astro.fetchContent("./posts/*.md");
```
The error did not happen after this change:
```
let allPosts = Astro.fetchContent("./posts/*.md");
const title = "Someone's blog"
```

**Comment by tusamni** [comment]
As others have noted, still getting this error on 0.20.12.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
